src/pricing.njk

```diff
@@ -4,46 +4,114 @@ align: center
 layout: layouts/page.njk
 sitemapPriority: 0.90
 meta:
-    description: Find out how much it costs to use FlowForge.
+description: Find out how much it costs to use FlowForge.
 ---
 
-<div class="px-6 pricing-tiles flex flex-col md:grid mb-16 md:max-w-screen-lg">
-    <div class="pricing-tile pt-4 pb-6 px-8">
-        <div class="flex flex-col justify-between items-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block">
-            <h4>Open-Source</h4>
-            <div class="mt-4">
-                <h2>$0</h2>
-                <label class="sub-label">Apache 2.0 Licensed<label>
-            </div>
-            <p class="mt-4">We are an open-core company, use FlowForge for free, always, with deployment options including Docker and Kubernetes.</p>
-            <a class="ff-btn ff-btn--secondary uppercase" href="/docs/install" onclick="capture('cta-install', {'position': 'primary'})">Install now</a>
-        </div>
+<div class="px-6 pricing-tiles flex flex-col mb-16 md:max-w-screen-lg">
+    <div class="flex px-6 -mb-6 -mt-32 mx-auto text-gray-400 md:max-w-screen-lg md:-mt-40 md:mb-1 md:block">
+        <p class="lead-p max-w-screen-lg m-auto text-center md:max-w-xl md:px-4">
+            Plans for every project. Whether you're an individual contributor, an IIoT manager, or an IT department, you'll find a suitable option to take your Node-RED applications to the next level.
+        </p>
     </div>
-    <div class="pricing-tile pt-4 pb-6 px-8">
-        <div class="flex flex-col justify-between items-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block">
-            <h4>Premium</h4>
-            <div>
-                <h2>$15</h2>
-                <label class="sub-label">per Node-RED instance/month</label>
+    <div class="pricing-tiles flex flex-col md:grid md:flex-row md:max-w-screen-lg">
+        <div class="pricing-tile pt-8 pb-6 md:h-full">
+            <div class="flex flex-col justify-between items-center text-center mb-4 sm:px-4 md:flex-row md:-mb-0 md:px-0 md:block md:text-left md:i md:items-left">
+                <h4 class="mt-2 pt-4" style="display:inline-block;">Starter </h4> <h5 style="display:inline-block;">(Open Source)</h5>
+                <label class="label">Self-managed</label>
+                <div class="mt-6 items-center md:text-left md:items-left">
+                    <h2>Free</h2>
+                </div>
+                <p class="mt-6 pb-2"><span class="inline font-normal">Open-Source</span> version of the platform, with deployment options including Docker and Kubernetes.</p>
+                </div>
+                <div class="flex flex-col items-center">
+                <a class="ff-btn ff-btn--secondary-outlined uppercase align-baseline w-full" href="/docs/install" onclick="capture('cta-install', {'position': 'primary'})">Install now</a>  
```

Comment:
This link is broken, maybe due to Netlify?

Reply:
I'm not sure, is the same link we have right now on main... And it works locally, for me at least...

Reply:
Probably Netlify not deploying the docs. Weird stuff